requirements.txt

```
@@ -1,3 +1,4 @@
 chardet==3.0.4
 karton-core>=5.0.0,<6.0.0
 python-magic==0.4.18
+yara-python
```

do we want to add a version restriction here?

---

I think we may need it, so it won't break randomly when yara-python updates. Maybe
```suggestion
yara-python>=4.0.0,<5.0.0
```